src/mame/drivers/alpha68k_n.cpp

```diff
@@ -758,16 +758,15 @@ void sstingray_state::sstingry(machine_config &config)
 	GFXDECODE(config, m_gfxdecode, m_palette, gfx_sstingry);
 	video_config(config, 0x40, 0, true);
 
-	/* sound hardware */
-
-	ym2203_device &ym1(YM2203(config, "ym1", 3000000));
-	ym1.add_route(ALL_OUTPUTS, "speaker", 0.30);
+	// sound hardware
+	ym2203_device &ym(YM2203(config, "ym", 2000000));            // Verified from video by PCB, 24MHz/12?
+	ym.add_route(ALL_OUTPUTS, "speaker", 0.30);
 
-	ym2203_device &ym2(YM2203(config, "ym2", 3000000));
-	ym2.add_route(ALL_OUTPUTS, "speaker", 0.30);
+	ay8910_device &aysnd1(AY8910(config, "aysnd1", 2000000));    // Verified from video by PCB, 24MHz/12?
+	aysnd1.add_route(ALL_OUTPUTS, "speaker", 0.30);
 
-	ym2203_device &ym3(YM2203(config, "ym3", 3000000));
-	ym3.add_route(ALL_OUTPUTS, "speaker", 0.45);
+	ay8910_device &aysnd2(AY8910(config, "aysnd2", 2000000));    // Verified from video by PCB, 24MHz/12?
+	aysnd2.add_route(ALL_OUTPUTS, "speaker", 0.45);
```

Comment:
Please use digit grouping so it’s easy to see order of magnitude at a glance without having to count the zeroes.